src/main/java/de/zalando/aruha/nakadi/repository/db/EventTypeCache.java

```diff
@@ -76,7 +86,7 @@ private void initParentCacheZNode(final CuratorFramework zkClient) throws Except
         }
     }
 
-    private void addCacheChangeListener(final LoadingCache<String, EventType> cache, final PathChildrenCache cacheSync) {
+    private void addCacheChangeListener(final LoadingCache<String, EventType> eventTypeCache, LoadingCache<String, EventTypeValidator> validatorCache, final PathChildrenCache cacheSync) {
```

Comment:
`With great power comes great responsibility`. If we don't use Jalopy - we should care about code formatting ourselves. Please try to avoid such long lines.